llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -305,14 +305,16 @@ void Loc::MMI::addFrameIndexExpr(const DIExpression *Expr, int FI) {
 
 static AccelTableKind computeAccelTableKind(unsigned DwarfVersion,
                                             bool GenerateTypeUnits,
+                                            bool HasSplitDwarf,
                                             DebuggerKind Tuning,
                                             const Triple &TT) {
   // Honor an explicit request.
   if (AccelTables != AccelTableKind::Default)
     return AccelTables;
 
   // Accelerator tables with type units are currently not supported.
-  if (GenerateTypeUnits)
+  if (GenerateTypeUnits &&
+      (DwarfVersion < 5 || HasSplitDwarf || !TT.isOSBinFormatELF()))
```

Comment:
Comment probably needs updating - and I'm not sure I understand why these three conditions are the right ones - so maybe the comment will help me understand better what's being added/handled here.

Reply:
more descriptive?

Reply:
Helpful, but I'm still curious why this is only supported for ELF at the moment.